manager/src/test/scala/org/broadinstitute/transporter/transfer/TransferListenerSpec.scala

```diff
@@ -315,10 +315,10 @@ class TransferListenerSpec extends PostgresSpec with MockFactory with EitherValu
 
     for {
       _ <- ids.traverse_ { id =>
-        sql"update transfers set status = 'submitted' where id = $id".update.run
+        sql"UPDATE transfers SET status = 'submitted' WHERE id = $id".update.run
       }.transact(tx)
       _ <- List(doUpdate, doRecord, doRecord, doUpdate, doRecord, doUpdate).parSequence_
-      statuses <- sql"select count(1) from transfers where status = 'succeeded'"
+      statuses <- sql"SELECT COUNT(1) FROM transfers WHERE status = 'succeeded'"
```

Comment:
unrelated to this refactor, but: I think the 'succeeded' and any other places where the status might be written out as the literal here should instead refer to the right `TransferStatus`